Counter/project.json

```diff
@@ -16,12 +16,23 @@
         "Microsoft.ServiceFabric": "5.0.135",
         "Microsoft.ServiceFabric.AspNetCore.Hosting": "1.0.0-*",
         "Microsoft.ServiceFabric.Services": "2.0.135"
+      },
+      "frameworkAssemblies": {
+        "System.Runtime": { "type": "build" },
+        "System.ComponentModel": { "type": "build" }
       }
     }
   },
   "runtimes": {
     "win7-x64": { }
   },
+
+  "runtimeOptions": {
+    "configProperties": {
+      "System.GC.Server": true
+    }
+  },
+
```

Comment:
It defaults to false in practice and is being added to some projects https://github.com/aspnet/benchmarks/commit/b7632666c7aa9a90f1893f5c677239d1795001b8